### DISCUSSION TITLE
Show thread pitch value in hole parameters panel

### DISCUSSION THREAD

**Discussion by RayTracedObserver** [FreeCAD/FreeCAD#15089] [Discussions that will turn in to issues but need refining]
Currently, when one is making a threaded hole using the hole tool from part design, one has to choose between regular or fine profiles. This determines the thread pitch for each hole size. However in order to know for sure which thread pitch is being modeled one has to look up the thread pitch values for coarse and fine (which are called regular and fine in freecad) either online or in a book or wherever they are written down.

It would be really useful if the hole parameter panel displayed the value of the thread pitch associated with each hole size and profile so that looking to an external reference is unnecessary. Unfortunately I do not know how to code apps... But maybe this is an easy to implement feature ?

**Comment by luzpaz** [comment]
What version are you using ? Full [About](https://wiki.freecad.org/About) info

**Reply by RayTracedObserver** [reply]
OS: macOS 14.5
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.38375 (Git)
Build type: Release
Branch: main
Hash: fe702349e848571802c01659668cc4bee519f4c6
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: C/Default (C)

**Reply by luzpaz** [reply]
Feel free to open a ticket if you can't find a similar feature request

**Reply by RayTracedObserver** [reply]
Alright I'll do so :)
